### ISSUE TITLE
Changing backpack material mostly to 'synthetic fabric'

### ISSUE THREAD

**Issue by metalheimer** [Coolthulhu/Cataclysm-DDA#2463]
Synthetic fabric (synfab). ID name "nylon". The material has existed and has been defined for a long time but it's basically unused. I've edited backpacks for myself to use synfab and been playing for dozens of hours. It bothered me that many backpacks were either cotton or leather, both of which are not ideal for rough and constant outdoor life. To me synfab for backpacks mean ripstop nylon or cordura, and for windbreaker jackets just lightweight polyester fabric. I've had no problems with the changes I've made.

The changes I've made:

-Changed many backpack crafting recipes to use synfab, while removing leather and/or cotton.

-Removed cotton as their material because I didn't want them to be repairable with cotton. Many backpacks are now a combination of synfab, leather and plastic.

-Changed windbreaker and emergency jacket material to synfab as well.

-Capped backpack item volume (NOT carry capacity) to be max 10L. This is because when an item is cut up, the amount materials you get is determined by its volume. For example, the "high-volume rucksack" is made of leather (!), it takes 14L of space, and if you cut it up, you'll get a mountain of leather. I'm thinking of reducing volumes further down to 5L because while it is true that an empty backpack takes space, it could also be rationalized a backpack itself be used as storage space for other items when not in use. If anything, a backpack can fill the gaps. Plus, I wanted to reduce the material yield.

-Maxed coverage to 30% because I was getting annoyed by backpacks being damaged by zombies. Might reduce it to 1%. Frankly backpacks being damaged needs some work, because a lone zombie shouldn't be able to damage it, and I don't want to protect myself with my backpack, heh. I'm dreaming of a more in-depth mechanic in which the backpack is basically immune to a single adjacent zombie in melee. But if there are two zombies on the opposite sides of the player in melee, then it should be just as susceptible as any clothing item. In short, the more zombies in melee, the more vulnerable the backpack should become. This is a whole other discussion, and I'm not able to pull off a change like this.

Overall, the way backpacks work is kind of retarded and out of user's control. Their protection values are determined by a value called "material thickness", so you can't define their protection values directly. Luckily coverage can be edited.

I'm asking: With multiple changes like these, would it be merged? Can it be safely tested by others? Can the changes be discarded if something unexpected comes up? Should I make this a mod instead?

### Describe the solution you'd like

see above

**Comment by chaosvolt** [comment]
If https://github.com/cataclysmbnteam/Cataclysm-BN/pull/2142 gets merged it'll fix the issue about backpack salvage varying by contents.

**Comment by Coolthulhu** [comment]
> Overall, the way backpacks work is kind of retarded and out of user's control. Their protection values are determined by a value called "material thickness", so you can't define their protection values directly.

Yeah, we should probably obsolete this weird thickness thing and go for direct armor definitions instead.

I'd also like to remove protection and damage chance from containers. Bags being ripped up isn't fun.
